documentation/src/docs/beginner_dev_guide.md

```diff
@@ -157,26 +157,68 @@ With this `streamConfig`, we can create streams that feel a bit more like tradit
 URI controllerURI = URI.create("tcp://localhost:9090");
 ```
 The code to create a Pravega Stream is as follows.
-```java
-try (StreamManager streamManager = StreamManager.create(controllerURI)) {
-        streamManager.createScope("examples");
-        streamManager.createStream("examples", "helloStream", streamConfig);
-}
-```
+
+=== "Java"
+
+    ```java
+    try (StreamManager streamManager = StreamManager.create(controllerURI)) {
+    streamManager.createScope("examples");
```

Comment:
As you have removed the indentation, is it being rendered in the generated documentation or not?

Reply:
I have the same question. I see these blocks "=== "Python"" in the generated md documentation.   is that right?